### DISCUSSION TITLE
Android Studio support

### DISCUSSION THREAD

**Discussion by agavrel** [rust-lang/rust-analyzer#15351] [Q&A]
Hi,

I love your work for Visual Studio but it is quite consuming to have 2 IDEs at the same time (using Android Studio for the front-end part), is there any support for Android Studio or project to do so?

**Comment by flodiebold** [comment]
Android Studio is based on the IntelliJ platform AFAIK, so if anything you might be able to install IntelliJ Rust, but I don't know if that works. We only officially provide the VSCode plugin and that only because we need one "reference" client; there are no plans to officially support any other plugins.

**Comment by agavrel** [comment]
Thanks, amazing work anyway!
